nftables.go

```diff
@@ -113,29 +133,42 @@ func (n *nft) Init() error {
 		}
 		n.set6 = set
 
-		n.conn6.AddRule(&nftables.Rule{
+		r := &nftables.Rule{
 			Table: n.table6,
 			Chain: chain,
-			Exprs: []expr.Any{
-				// [ payload load 4b @ network header + 16 => reg 1 ]
-				&expr.Payload{
-					DestRegister: 1,
-					Base:         expr.PayloadBaseNetworkHeader,
-					Offset:       8,
-					Len:          16,
-				},
-				// [ lookup reg 1 set whitelist ]
-				&expr.Lookup{
-					SourceRegister: 1,
-					SetName:        n.set6.Name,
-					SetID:          n.set6.ID,
-				},
-				//[ immediate reg 0 drop ]
-				&expr.Verdict{
-					Kind: expr.VerdictDrop,
-				},
-			},
+			Exprs: []expr.Any{},
+		}
+		// [ payload load 4b @ network header + 16 => reg 1 ]
+		r.Exprs = append(r.Exprs, &expr.Payload{
+			DestRegister: 1,
+			Base:         expr.PayloadBaseNetworkHeader,
+			Offset:       8,
+			Len:          16,
 		})
+		// [ lookup reg 1 set whitelist ]
+		r.Exprs = append(r.Exprs, &expr.Lookup{
+			SourceRegister: 1,
+			SetName:        n.set6.Name,
+			SetID:          n.set6.ID,
+		})
+		if n.DenyLog {
+			r.Exprs = append(r.Exprs, &expr.Log{
+				Key:  unix.NFTA_LOG_PREFIX,
+				Data: []byte(n.DenyLogPrefix),
+			})
+		}
+    if strings.EqualFold(n.DenyAction, "REJECT") {
```

Comment:
i think there is an indentation problem here also

Reply:
yes, should be fixed now